src/detectors/builtin/suspiciousMessageMode.ts

```diff
@@ -72,6 +72,20 @@ export class SuspiciousMessageMode extends ASTDetector {
     expr: AstExpression,
     warnings: MistiTactWarning[],
   ): void {
+    if (expr.kind === "number" && expr.value === 0n) {
+      warnings.push(
```

Comment:
Let's replace it with the new constant name introduced in https://github.com/tact-lang/tact/issues/988

Reply:
We will add this when tact lang add it.

Reply:
It has been merged. Now, we can advice to use the `SendDefaultMode` constant instead.